ec/ecresource/deploymentresource/apm/v2/apm_read.go

```diff
@@ -95,3 +99,13 @@ func IsApmStopped(res *models.ApmResourceInfo) bool {
 	return res == nil || res.Info == nil || res.Info.Status == nil ||
 		*res.Info.Status == "stopped"
 }
+
+func SetLatestInstanceConfigInfo(currentTopology *Apm, latestTopology *models.ApmTopologyElement) {
+	if currentTopology != nil && latestTopology != nil {
+		currentTopology.LatestInstanceConfigurationId = &latestTopology.InstanceConfigurationID
+		if latestTopology.InstanceConfigurationVersion != nil {
+			latestVersion := int(*latestTopology.InstanceConfigurationVersion)
+			currentTopology.LatestInstanceConfigurationVersion = &latestVersion
```

Comment:
```suggestion
			currentTopology.LatestInstanceConfigurationVersion = ec.Int(int(*latestTopology.InstanceConfigurationVersion))
```
Actually you did something like that in other places. 
It could be that the suggestion cannot be used as is without importing "github.com/elastic/cloud-sdk-go/pkg/util/ec".

Reply:
You're totally right, thanks for the suggestion! Updated in https://github.com/elastic/terraform-provider-ec/pull/755/commits/4b95ac5b2f6dbb1600bb93644cf3335007cc6a3f.